var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -69,6 +69,8 @@ def setup_build_environment(self, env):
             env.append_flags('CFLAGS', self.compiler.cc_pic_flag)
         if '+optimize' in self.spec:
             env.append_flags('CFLAGS', '-O2')
+        if '+notpossible' in self.spec:
+            env.append_flags('CFLAGS', '-Wall')
```

Comment:
Did you mean to include this change in the PR?

Reply:
It's a separate commit with WIP in the message.  Just to trick gitlab into scheduling some jobs, since otherwise now we prune jobs that are not "touched" (meaning their package.py doesn't appear in the PR, and neither does any transitive dependency or depdendent.)  I'll remove the WIP commit once we see gitlab do the right thing.